src/API.ts

```diff
@@ -343,7 +347,7 @@ export type FunctionWithSameArgs<F extends AnyFunction> = (...args: Parameters<F
  * @interface Shell
  * @extends {(Pick<AppHost, Exclude<keyof AppHost, 'getStore' | 'log' | 'options'>>)}
  */
-export interface Shell extends Pick<AppHost, Exclude<keyof AppHost, 'getStore' | 'log' | 'options'>> {
+export interface Shell extends Pick<AppHost, Exclude<keyof AppHost, 'getStore' | 'log' | 'options' | 'isInstallingEntryPoints'>> {
```

Comment:
I would add getStore, log and options to the privateAppHost and exclude all of it
We can do it in a different pr